nix/default.nix

```diff
@@ -0,0 +1,84 @@
+{
+  pkgs ? import <nixpkgs> { },
+  src ? ../.,
+  targets ? ["all"],
+  # Options: 0,1,2
+  verbosity ? 0,
+  # Use system Nim compiler instead of building it with nimbus-build-system
+  useSystemNim ? true,
+  commit ? builtins.substring 0 7 (src.rev or "dirty"),
+  # These are the only platforms tested in CI and considered stable.
+  stableSystems ? [
+    "x86_64-linux" "aarch64-linux"
+    "x86_64-darwin" "aarch64-darwin"
+  ],
+  circomCompatPkg ? (
+    builtins.getFlake "github:codex-storage/circom-compat-ffi"
+  ).packages.${builtins.currentSystem}.default
+}:
+
+let
+  inherit (pkgs) stdenv lib writeScriptBin callPackage;
```

Comment:
You're not actually using the `callPackage` from here.

Reply:
I am using it in line 25

Reply:
Do you mean here?
```
        buildTarget = pkgsFor.${system}.callPackage ./nix/default.nix {
```
No you don't. That's a different `callPackage`.

Reply:
I meant here:
```nix
tools = pkgs.callPackage ./tools.nix {};
```
but I should get rid of `pkgs` prefix here